app/views/ui-components/bs4/v1/navs/_header.html.erb

```diff
@@ -5,46 +5,99 @@
     <a class="navbar-brand mr-0" href="#">
       <%= image_pack_tag "logo_bs4_#{ENV['CLIENT'] || 'ic'}.svg", alt: "#{ENV['CLIENT']} logo"%>
     </a>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#nav-dynamic-collapse" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"><%= image_pack_tag "icons/menu-burger.svg", class:"h-100", alt:"navbar collapse burger" %></span>
+    <button class="navbar-toggler unstyled collapsed" type="button" data-toggle="collapse" data-target="#nav-dynamic-collapse" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon open">
+        <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
might need a title/alt for the svgs for accessibility

Reply:
done, thanks for the reminder!